<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Select_Truck_2</class>
 <widget class="QDialog" name="Select_Truck_2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>955</width>
    <height>679</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1263</width>
     <height>751</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>232</red>
         <green>241</green>
         <blue>252</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Light">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>232</red>
         <green>241</green>
         <blue>252</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>232</red>
         <green>241</green>
         <blue>252</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>232</red>
         <green>241</green>
         <blue>252</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="NoRole">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>232</red>
         <green>241</green>
         <blue>252</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>232</red>
         <green>241</green>
         <blue>252</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Light">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>232</red>
         <green>241</green>
         <blue>252</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>232</red>
         <green>241</green>
         <blue>252</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>232</red>
         <green>241</green>
         <blue>252</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="NoRole">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>232</red>
         <green>241</green>
         <blue>252</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>232</red>
         <green>241</green>
         <blue>252</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Light">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>232</red>
         <green>241</green>
         <blue>252</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>232</red>
         <green>241</green>
         <blue>252</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>232</red>
         <green>241</green>
         <blue>252</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="NoRole">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>232</red>
         <green>241</green>
         <blue>252</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="styleSheet">
    <string notr="true">
/**********页签项**********/
QTabWidget::pane {
		border: none;
		border-top: 3px solid rgb(0, 78, 161);
	
	background-color: rgb(232, 241, 252);

}
QTabWidget::tab-bar {
		border: none;
}
QTabBar::tab {
		border: none;
		border-top-left-radius: 4px;
		border-top-right-radius: 4px;
		color: white;
		background: rgb(120, 170, 220);
		height: 28px;
		min-width: 85px;
		margin-right: 5px;
		padding-left: 5px;
		padding-right: 5px;
}
QTabBar::tab:hover {
		background: rgb(0, 78, 161);
}
QTabBar::tab:selected {
		color: white;
		background: rgb(0, 78, 161);
}

QTabWidget,tabWidget::pane {
        border: 1px solid rgb(111, 156, 207);
        background: rgb(232, 241, 252);
        margin-top: -1px;
}

QTabBar,tabBar::tab {
        border: 1px solid rgb(111, 156, 207);
        border-bottom: none;
        color: rgb(70, 71, 73);
        background: transparent;
}
QTabBar,tabBar::tab:hover {
        color: rgb(2, 65, 132);
}
QTabBar,tabBar::tab:selected {
        color: rgb(2, 65, 132);
        background: rgb(232, 241, 252);
}</string>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="tab_28">
    <attribute name="title">
     <string>查询卡车信息</string>
    </attribute>
    <widget class="QComboBox" name="combo_Box_Select">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>111</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>新宋体</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(94, 105, 255);</string>
     </property>
     <item>
      <property name="text">
       <string>按编号</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>按车牌号</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>按制造公司</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>按购买时间</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="line_Select_Message">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>50</y>
       <width>521</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>华文楷体</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit
{
    background-color: rgb(134,183,200);
    border:2px solid #5F92B2;
    border-radius:3px;
}




</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>请输入需要查询的信息</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_Select">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>40</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>新宋体</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">
QPushButton
{
    background-color: rgb(134,183,200);
    border:2px solid rgb(0,255,255);
    border-radius:5px;
    
	
	color: rgb(0, 255, 255);
}


QPushButton:hover
{
    background-color: rgb(0,130,150);
    border:2px solid #5F92B2;
    border-radius:5px;
    color:white;
}

QPushButton:hover:pressed
{
    background-color:rgb(85,170,255);
    border:2px solid #3C80B1;
    border-radius:5px;
    color:white;
}</string>
     </property>
     <property name="text">
      <string>查询</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>0</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>华文楷体</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(94, 105, 255);</string>
     </property>
     <property name="text">
      <string>查询卡车信息</string>
     </property>
    </widget>
    <widget class="QTableView" name="table_Message">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>921</width>
       <height>511</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_29">
    <attribute name="title">
     <string>查询小轿车信息</string>
    </attribute>
    <widget class="QTableView" name="table_Message_21">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>921</width>
       <height>511</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
    <widget class="QComboBox" name="combo_Box_Select_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>111</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>新宋体</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(94, 105, 255);</string>
     </property>
     <item>
      <property name="text">
       <string>按编号</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>按车牌号</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>按制造公司</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>按购买时间</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="line_Select_Message_2">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>50</y>
       <width>521</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>华文楷体</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit
{
    background-color: rgb(134,183,200);
    border:2px solid #5F92B2;
    border-radius:3px;
}




</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>请输入需要查询的信息</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_206">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>0</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>华文楷体</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(94, 105, 255);</string>
     </property>
     <property name="text">
      <string>查询小轿车信息</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_Select_2">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>40</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>新宋体</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">
QPushButton
{
    background-color: rgb(134,183,200);
    border:2px solid rgb(0,255,255);
    border-radius:5px;
    
	
	color: rgb(0, 255, 255);
}


QPushButton:hover
{
    background-color: rgb(0,130,150);
    border:2px solid #5F92B2;
    border-radius:5px;
    color:white;
}

QPushButton:hover:pressed
{
    background-color:rgb(85,170,255);
    border:2px solid #3C80B1;
    border-radius:5px;
    color:white;
}</string>
     </property>
     <property name="text">
      <string>查询</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_30">
    <attribute name="title">
     <string>查询大客车信息</string>
    </attribute>
    <widget class="QLabel" name="label_210">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>0</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>华文楷体</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(94, 105, 255);</string>
     </property>
     <property name="text">
      <string>查询大客车信息</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="line_Select_Message_3">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>50</y>
       <width>521</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>华文楷体</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit
{
    background-color: rgb(134,183,200);
    border:2px solid #5F92B2;
    border-radius:3px;
}




</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>请输入需要查询的信息</string>
     </property>
    </widget>
    <widget class="QComboBox" name="combo_Box_Select_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>111</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>新宋体</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(94, 105, 255);</string>
     </property>
     <item>
      <property name="text">
       <string>按编号</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>按车牌号</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>按制造公司</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>按购买时间</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="btn_Select_3">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>40</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>新宋体</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">
QPushButton
{
    background-color: rgb(134,183,200);
    border:2px solid rgb(0,255,255);
    border-radius:5px;
    
	
	color: rgb(0, 255, 255);
}


QPushButton:hover
{
    background-color: rgb(0,130,150);
    border:2px solid #5F92B2;
    border-radius:5px;
    color:white;
}

QPushButton:hover:pressed
{
    background-color:rgb(85,170,255);
    border:2px solid #3C80B1;
    border-radius:5px;
    color:white;
}</string>
     </property>
     <property name="text">
      <string>查询</string>
     </property>
    </widget>
    <widget class="QTableView" name="table_Message_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>921</width>
       <height>511</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
